addons/mass_mailing/static/src/scss/mass_mailing_mobile_preview.scss

```diff
@@ -18,7 +18,7 @@
                 font-family: $o-we-font-family;
 
                 .btn-close {
-                    filter: invert(1) grayscale(100%) brightness(200%);
+                    filter: invert(50%) grayscale(100%) brightness(200%);
```

Comment:
LGTM, just commit message header I would update, like "be sure cross is displayed in dark mode" ?